src/compiler/binder.ts

```diff
@@ -2478,6 +2478,13 @@ namespace ts {
             declareSymbol(symbolTable, symbol, propertyAccess, symbolFlags, symbolExcludes);
         }
 
+        function getParentOfBinaryExpression(expr: BinaryExpression) {

```

Comment:
is there a reason why this function duplicated?

Reply:
I introduced `getParentOfBinaryExpression` and `getInitializerOfBinaryExpression` in this PR. Are you confusing those two?